src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,27 @@
+package racingcar;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarTest {
+
+    @DisplayName("자동차가 생성된다")
+    @Test
+    void createCar() {
+        Car car = new Car();
+        assertThat(car).isNotNull();
+    }
+
+    @DisplayName("값이 4 이상일 때 자돋차가 이동한다")
+    @ParameterizedTest
+    @CsvSource({"0,0", "3,0", "4,1", "5,1"})
```

Comment:
경계값 기준으로 테스트 굿

```suggestion
    @CsvSource({"0,0", "3,0", "4,1", "9,1"})
```
위와 같이 구현하면 더 확실한 경계값 테스트가 되지 않을까?